src/components/families/family-sidebar/previous-enrolment-card/PreviousEnrolmentCard.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useEffect } from "react";
+
+import { Box, Divider, IconButton, Typography } from "@material-ui/core";
+import Collapse from "@material-ui/core/Collapse";
```

Comment:
is it possible to combine these imports? It seems they're both from material-ui/core?